pkg/querier/worker/scheduler_processor.go

```diff
@@ -49,6 +54,8 @@ func newSchedulerProcessor(cfg Config, handler RequestHandler, log log.Logger, r
 		maxMessageSize: cfg.GRPCClientConfig.MaxSendMsgSize,
 		querierID:      cfg.QuerierID,
 		grpcConfig:     cfg.GRPCClientConfig,
+		//
```

Comment:
Remove or comment :slightly_smiling_face: 

```suggestion
		//
```